src/project/server/routes/api.js

```diff
@@ -0,0 +1,114 @@
+// Define your endpoints here (this is your "controller file")
+const express = require('express');
+const ItemManager = require('../services/item-manager.js')
+
+const todoRouter = express.Router();
+const itemManager = new ItemManager();
+
+todoRouter.get('/', async(req, res) => {
+    const {sort, filter} = req.query;
+    
+    let sortedList = [];
+
+    const sortObj = {
```

Comment:
this should be in a const file and with the right casing, no need to redefine this every time the endpoint is called